app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("Vaccine is missing")
+        else:
```

Comment:
The use of `else` after a `raise` statement is unnecessary here. According to the checklist, avoid using unnecessary `else` statements. You can directly assign `vaccine_date` after the `raise` statement without using `else`.